doc/classes/Label.xml

```diff
@@ -62,8 +62,8 @@
 		</member>
 		<member name="mouse_filter" type="int" setter="set_mouse_filter" getter="get_mouse_filter" overrides="Control" enum="Control.MouseFilter" default="2" />
 		<member name="percent_visible" type="float" setter="set_percent_visible" getter="get_percent_visible" default="1.0">
-			Limits the number of visible characters. If you set [code]percent_visible[/code] to 0.5, only up to half of the text's characters will display on screen. Useful to animate the text in a dialog box.
-			[b]Note:[/b] Setting this property updates [member visible_characters] based on current [method get_total_character_count].
+			The number of characters to display, in relation to the total character count (see [method get_total_character_count]). If [code]1.0[/code], all characters display. If [code]0.5[/code], only up to half the characters will display.
```

Comment:
I don't like starting the description with "the number of characters", when it's not. I think people may misread that, because to me it strongly implies a concrete measure in units. Since I'm not a native speaker, I may be wrong, but would the word "amount" sound better, since it doesn't imply any units?

Alternatively, try something more applicable here, like portion or fraction.

I also don't mind leaving an example in, telling that this is useful for progressively displaying text with an animation, like a dialog box for example. It's good to have small bits and pointers like that, IMO.

Reply:
>  I may be wrong, but would the word "amount" sound better, since it doesn't imply any units?

I remember this https://github.com/godotengine/godot/pull/64551 being merged last time. It was the one, in fact, that changed the "_Limits the **amount** of visible characters_" of the description to "_Limits the **number**  of visible characters_". So I wouldn't reintroduce "amount" again in this context. I can try "fraction"...

> I also don't mind leaving an example in, telling that this is useful for progressively displaying text with an animation,

In my (small) experience, I've found `visible_characters` to be more useful for animating dialogue, as it's a simpler matter of adding 1 and, say, playing a sound, at the same rate, regardless of text length.
I didn't want to encourage using `visible_percent` for it, but it's a good point. It doesn't hurt to put the example back.

Reply:
> I remember this #64551 being merged last time. It was the one, in fact, that changed the "_Limits the **amount** of visible characters_" of the description to "_Limits the **number** of visible characters_". So I wouldn't reintroduce "amount" again in this context.

Right, but the meaning if different now. Saying that it limits the number is correct, but saying that it is the number is not. Maybe @asmaloney can weigh in on that.

Reply:
Changed it up now to fraction, I think it sounds better this way, too.

Reply:
I agree the meaning changed somewhat. Looking at the current version:

> The fraction of characters to display, related to the total character count (see [method get_total_character_count]).

I would say it's still a bit awkward.

Maybe something like (since percent is a specific fraction and that's what we're dealing with):

> Percentage of the total character count to display (see [method get_total_character_count]).

I would also suggest adding an example because sometimes the naming using percentage is ambiguous: is `0.38` 0.38% or 38%?

Reply:
> I would also suggest adding an example because sometimes the naming using percentage is ambiguous: is 0.38 0.38% or 38%?

Hmm... Yeah, hence the other PR... 
Isn't the "_If [code]0.5[/code], only up to half the characters will display_" a meaningful example for the percentage already ("0.5 equals "half" thus "50%")?

Reply:
Oh - so sorry. I didn't continue reading... 

My only excuse is that it was pre-coffee for me.  😄